package.json

```diff
@@ -18,24 +18,23 @@
     "build:framework": "yarn --cwd ./akasha-core build:all"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-beta.20",
-    "@docusaurus/preset-classic": "2.0.0-beta.20",
+    "@docusaurus/core": "^2.4.1",
+    "@docusaurus/preset-classic": "^2.4.1",
     "@mdx-js/react": "^1.6.22",
     "clsx": "^1.1.1",
     "prism-react-renderer": "^1.2.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1"
-
   },
   "devDependencies": {
-    "@docusaurus/module-type-aliases": "2.0.0-beta.17",
+    "@docusaurus/module-type-aliases": "^2.4.1",
     "@tsconfig/docusaurus": "^1.0.4",
     "docusaurus-plugin-includes": "^1.1.4",
-    "typescript": "^4.6.2",
-    "typedoc": "0.22.15",
     "docusaurus-plugin-typedoc": "0.17.5",
+    "typedoc": "0.22.15",
```

Comment:
update typedoc also to `0.24.8`

Reply:
and also all the typedoc and docusaurus plugins to their respective last version

Reply:
Thanks Marius. I have updated them :)